app/models/template_version.rb

```diff
@@ -20,6 +20,17 @@ def lookup_props
     flatten_requirements_from_form_hash(form_json)
   end
 
+  def form_json_requirements
+    requirement_ids = []
+    form_json["components"].each do |section|
+      section["components"].each do |requirement_block|
+        requirement_block["components"].each { |requirement| requirement_ids.push(requirement["id"]) }
+      end
+    end
```

Comment:
please use the ` requirement_block_json` for this. Avoid using `form_json` as it's in the formio format, and it can change